src/fundus/publishers/de/motorsport_magazin.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class MotorSportMagazinParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector: CSSSelector = CSSSelector("section.article-body")
+        _subheadline_selector: CSSSelector = CSSSelector("p.teaser")
+
+        @attribute
```

Comment:
Could you also add a function for the topics? Motorsport Magazin supports the meta `news_keywords` tag which could be used for that purpose.

Reply:
also added! :)